doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -169,6 +169,115 @@ Create OpenLayers application
 
 These tiles are being rendered by the OpenLayers client.
 
+Create OpenLayers application - WMS Vector Tiles
+------------------------------------------------
+
+.. note::   Vector tiles requested with WMS allows to retrieve non-cached vector tiles (server side), setting the ``tiled=false`` parameter on the ``getMap`` request. This setting could be particularly useful when serving fast changing source data that should constantly be kept up-to-date for display.
+
+#. Create a ``www/wms-vectortiles`` directory inside your GeoServer Data Directory.
+
+#. Download the `latest version of OpenLayers <http://openlayers.org/download/>`_.
+
+#. Extract the following files to from the downloaded archive to the directory created in step 1:
+
+   * ``ol.js``
+   * ``ol-debug.js``
+   * ``ol.css``
+
+#. In a text editor, create a new file with the following content:
+
+   .. code-block:: html
+      :emphasize-lines: 5
+      
+      <!doctype html>
+      <html>
+      <head>
+        <title>Vector tiles</title>
+        <script src="ol.js"></script>
+        <link rel="stylesheet" href="ol.css">
+        <style>
+          html, body {
+            font-family: sans-serif;
+            width: 100%;
+          }
+          .map {
+            height: 500px;
+            width: 100%;
+          }
+        </style>
+      </head>
+        <body>
+          <h3>Mapbox Protobuf - vector tiles WMS</h3>
+          <div class="refresh-container">
+          <button id="refresh-button" type="button" onclick="updateFunc();">Refresh/reload cache</button>
+          </div>
+          <div id="map" class="map"></div>
+          <script>
+          
+            var layerParams = {'LAYERS': 'opengeo:countries', 'TILED': false, 'format': 'application/vnd.mapbox-vector-tile'};
```

Comment:
Something odd here, the generate URL ends up having "format" twice, once uppercase, once lowercase, e.g:

```
http://localhost:8080/geoserver/wms?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&FORMAT=image%2Fpng&TRANSPARENT=true&LAYERS=importer%3Ane_10m_roads&TILED=false&format=application%2Fvnd.mapbox-vector-tile&WIDTH=256&HEIGHT=256&CRS=EPSG%3A3857&STYLES=&BBOX=10018754.17139462%2C0%2C20037508.34278924%2C10018754.171394622
``

Reply:
@aaime yes 'FORMAT' had to be specified in upper case for the property to be overridden